framework/components/ecl-forms/ecl-forms-labels/ecl-forms-labels.twig

```diff
@@ -1 +1 @@
-<label for="example-input-id">Some Label</label>
\ No newline at end of file
+<label class="ecl-form-label" for="example-input-id">Some Label</label>
```

Comment:
the `for` attr is maybe not necessary in this case, or include a small input and [info](https://www.w3.org/WAI/tutorials/forms/labels/) to demonstrate the use.